rllib/evaluation/worker_set.py

```diff
@@ -397,6 +407,40 @@ def _from_existing(local_worker: RolloutWorker,
         workers._remote_workers = remote_workers or []
         return workers
 
+    def _get_dataset_and_shards(self, config: TrainerConfigDict,
+                                num_workers: int, local_worker: bool)\
+            -> (ray.data.dataset.Dataset,
+                List[ray.data.dataset.Dataset]):
+        assert config["input"] == "dataset"
+        assert "input_config" in config, (
+            "Must specify input_config dict if using Dataset input.")
+
+        input_config = config["input_config"]
+        if (not input_config.get("type", None) or
+            not input_config.get("path", None)):
+            raise ValueError(
+                "Must specify type and path via input_config key"
+                " when using Ray dataset input.")
+
+        type = input_config["type"]
+        path = input_config["path"]
+        if type == "json":
+            dataset = data.read_json(path)
+        elif type == "parquet":
+            dataset = data.read_parquet(path)
+        else:
+            raise ValueError("Un-supported Ray dataset type: ", type)
+
+        if local_worker and num_workers == 0:
+            # Local worker will be responsible for sampling.
```

Comment:
Nit: Can we move these comments on top of the `if` and `else`?

Reply:
this particular comment is for the case when we don't have remote worker though. so I don't really have to split the ds.

Reply:
oh, I see what you mean below, ok, done.